modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/impl/CassandraAssignmentsRepository.java

```diff
@@ -556,6 +552,196 @@ public Assignment assignUser(Assignment assignment, Experiment experiment, Date
         return new_assignment;
     }
 
+    /**
+     *
+     * Create use assignments in cassandra in a batch.
+     *
+     * @param assignments pair of experiment and assignment
+     * @param date       Date of user assignment
+     * @return
+     */
+    @Override
+    @Timed
+    public Assignment assignUser(List<Pair<Experiment, Assignment>> assignments, Date date) {
```

Comment:
I think explicitly naming this method to indicate this is a mutli-assignment may be more readable rather than overloading.